src/Page/Food/Builder.elm

```diff
@@ -760,6 +766,46 @@ packagingListView db selectedImpact recipe results =
     ]
 
 
+distributionView : Impact.Definition -> Recipe -> Recipe.Results -> List (Html Msg)
+distributionView selectedImpact recipe results =
+    [ div [ class "card-header d-flex align-items-center justify-content-between" ]
+        [ h5 [ class "mb-0" ] [ text "Stockage et distribution" ]
+        , Format.formatFoodSelectedImpact selectedImpact <| results.recipe.conservation
```

Comment:
This is because the forward-style data transformation is more easily readable when it looks like left-to-right pipe transformation (like in the shell).
I've reversed it in this example.